kw

```diff
@@ -50,17 +62,27 @@ if [ -f "${KW_BASE_DIR}/src/lib/kwlib.sh" ]; then
   KW_ETC_DIR="${KW_BASE_DIR}/etc"
   KW_PLUGINS_DIR="${KW_LIB_DIR}/plugins"
   # KW_CACHE_DIR # use default cache folder
-fi
-##END-DEV-MODE##
 
-# Export external variables required by kworkflow
-export KWORKFLOW
+  # include IO utilities
+  include "${KW_LIB_DIR}/lib/kwio.sh"
 
-#INJECT_CODE_TRACING_SETUP
+  # Print a notice to the developers so he knows some envvars have been changed
+  # We redirect to stderror because this output would make tests fails if the developer is testing
+  say "[INFO]: Running kw in developer mode. Envvars set to the following:
 
-# This is the one and only time a file will be sourced this way.
-# The include function (sourced from this file) should always be used for file sourcing.
-. "${KW_LIB_DIR}/lib/kw_include.sh" --source-only
+KW_CACHE_DIR =${KW_CACHE_DIR}
+KW_DATA_DIR=${KW_DATA_DIR}
+KW_DB_DIR=${KW_DB_DIR}
+KW_DOC_DIR=${KW_DOC_DIR}
+KW_ETC_DIR=${KW_ETC_DIR}
+KW_LIB_DIR=${KW_LIB_DIR}
+KW_MAN_DIR=${KW_MAN_DIR}
+KW_PLUGINS_DIR=${KW_PLUGINS_DIR}
+KW_SOUND_DIR=${KW_SOUND_DIR}
+" >&2
```

Comment:
Could explain to me the reason for using the std-err stream? 😅

Reply:
surely! 

If the developer is running in "repo mode", he/she will probably want to run the tests at some moment; sending the message `you are running kw in developer mode` will modify the expected output of kw in the tests and I thought the tests may fail.

As I said, I thought the tests may fail, but I just ran the tests here with the notice message sent to std-out and all tests passed... That means my guess was wrong. I'll use std-out instead of std-err

Reply:
Oh, I get it now! Thanks for the explanation, I confess I agree with your reasoning, but since the tests don't fail using std-out, its better to use it for clarity.